Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSPatchSensitivity.py

```diff
@@ -109,8 +109,8 @@ def __patch_workspace(self, tube_in_input_ws, in_ws, patch_ws):
                 id_to_fit.append(detector_id)
             elif not inputDetInfo.isMasked(detector_idx):
                 id_to_calculate_fit.append(detector_id)
```

Comment:
It's all OK! I've tested that with the users. It would be great if you could get this into the release `3.9`. Thanks! Ricardo